utils/dd_to_csv.py

```diff
@@ -154,7 +154,27 @@ def save_data_with_headers(
     return
 
 
-def convert_dd_to_tabular(basedir: str, output_dir: str) -> None:
+def generate_headers_by_attr() -> Dict[str, List[str]]:
+
+    with open("xl2times/config/times-info.json") as f:
+        attributes = f.read()
+
+    attributes = eval(attributes)
```

Comment:
In general this is a potential security risk: reading arbitrary python code from a file and executing it! It is safer to use `json.load` to read from a JSON file.

Reply:
Well, since the contents of the file is controlled by us, I thought it was not that much of a risk. :-)
The main advantage of `eval` over `json.load` for loading json files that I see is that it seems to be able to interpret the nested structure correctly. `json.load` stops nesting at some point and just interpretes the remaining nested structure as a string.

I agree that `json.load` works perfectly fine here. :-)